{"name": "验证错错误登录", "status": "passed", "attachments": [{"source": "81d8f77d-e727-4899-bab7-cb3531ae9ba7-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "9d927d36-26fd-428d-b9cb-f3f4d339054d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "'19100129893'"}, {"name": "pw", "value": "'000000'"}, {"name": "msg", "value": "'登录失败，验证码错误'"}], "start": 1622712895584, "stop": 1622712902515, "uuid": "122947c8-fdcf-49fc-b1e8-6f845ac72930", "historyId": "290471539cbdc286c700105d0ab76560", "testCaseId": "0a157dd46979acb51da04934dbb33d2c", "fullName": "test_login.TestLogin#test_sms_login_error", "labels": [{"name": "feature", "value": "用户登录模块"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MicroWin10-1243"}, {"name": "thread", "value": "6380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}